src/cmd/manage_nsfs.js

```diff
@@ -451,7 +453,7 @@ async function add_account(data) {
     nsfs_schema_utils.validate_account_schema(JSON.parse(data));
     await native_fs_utils.create_config_file(fs_context, accounts_dir_path, account_config_path, data);
     await native_fs_utils._create_path(access_keys_dir_path, fs_context, config.BASE_MODE_CONFIG_DIR);
-    await nb_native().fs.symlink(fs_context, account_config_path, account_config_access_key_path);
+    await nb_native().fs.symlink(fs_context, account_config_relative_path, account_config_access_key_path);
```

Comment:
These changes - defining the variable `account_config_relative_path` and `symlink` to this path -  currently only appear in `add_account`, what about `update_account`?

Reply:
Thanks, code updated